### PR TITLE
review cancer branch

### PR DESCRIPTION
**Pull request by sabrinatoro**
Addresses #6533
Changes based on expert review [here](https://docs.google.com/spreadsheets/d/1fXgb5tXZA-gtdFBsM3TQDK0cSk5RIxZypgRwF_1knnQ/edit#gid=0)

### CODE REVIEW

**Comment by sabrinatoro** [comment]
This issue also 
Closes #7460

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -335532,6 +335532,7 @@ xref: UMLS:C1096154 {source="Orphanet:1183/e", source="MONDO:notFoundInDiseaseSu
 xref: UMLS:C1721017 {source="MONDO:notFoundInDiseaseSubset", source="Orphanet:1183"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
 is_a: MONDO:0005395 {source="MONDO:0017652-obsoleted"} ! movement disorder
+is_a: MONDO:0018215 ! paraneoplastic neurologic syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:335535]: this should have a source

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -342527,6 +342528,7 @@ xref: Orphanet:163895 {source="MONDO:equivalentObsolete"}
 xref: SCTID:445014002 {source="MONDO:equivalentTo"}
 xref: UMLS:C0338430 {source="NCIT:C4350", source="MONDO:notFoundInDiseaseSubset", source="Orphanet:163895"}
 is_a: MONDO:0015588 {source="MONDO:Redundant", source="Orphanet:163895"} ! limbic encephalitis
+is_a: MONDO:0018215 ! paraneoplastic neurologic syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:342531]: add source

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -377266,7 +377270,9 @@ xref: UMLS:C0085404 {source="NCIT:C80303", source="Orphanet:2905/e", source="MON
 xref: UMLS:C1510415 {source="Orphanet:2905/e", source="MONDO:notFoundInDiseaseSubset", source="Orphanet:2905"}
 is_a: MONDO:0002254 {source="NCIT:C80303"} ! syndromic disease
 is_a: MONDO:0015923 {source="Orphanet:2905"} ! acquired peripheral neuropathy
+is_a: MONDO:0018215 ! paraneoplastic neurologic syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:377273]: add source

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -377266,7 +377270,9 @@ xref: UMLS:C0085404 {source="NCIT:C80303", source="Orphanet:2905/e", source="MON
...
 is_a: MONDO:0021073 {source="Orphanet:2905", source="https://orcid.org/0000-0001-5208-3432"} ! paraneoplastic syndrome
+is_a: MONDO:0700252 ! parneoplastic endocrine syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:377275]: add source

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -391776,6 +391787,8 @@ is_a: MONDO:0001068 {source="NCIT:C67235"} ! osteomalacia
 is_a: MONDO:0005151 {source="MONDO:0019061-obsoleted"} ! endocrine system disorder
 is_a: MONDO:0005381 {source="MONDO:Redundant", source="Orphanet:352540"} ! bone disorder
 is_a: MONDO:0021073 {source="NCIT:C67235"} ! paraneoplastic syndrome
+is_a: MONDO:0700255 ! paraneoplastic renal syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:391790]: add source

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -400371,6 +400384,7 @@ xref: SCTID:230688006 {source="DOID:0050214"}
 xref: SCTID:56989000 {source="DOID:0050214", source="MONDO:equivalentTo"}
 xref: UMLS:C0022972 {source="Orphanet:43393/e", source="DOID:0050214", source="MONDO:equivalentTo", source="NCIT:C3155", source="Orphanet:43393"}
 is_a: MONDO:0018215 {source="MONDO:Redundant", source="Orphanet:43393"} ! paraneoplastic neurologic syndrome
+is_a: MONDO:0018215 ! paraneoplastic neurologic syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:400387]: add source

File: src/ontology/mondo-edit.obo [hunk 7]
```diff
@@ -409236,6 +409250,8 @@ xref: Orphanet:63455 {source="MONDO:equivalentTo"}
 xref: UMLS:C1112570 {source="Orphanet:63455", source="MONDO:equivalentTo", source="Orphanet:63455/e"}
 is_a: MONDO:0019337 {source="EFO:0008602/inferred", source="Orphanet:63455"} ! autoimmune bullous skin disease
 is_a: MONDO:0021073 {source="https://orcid.org/0000-0002-6601-2165"} ! paraneoplastic syndrome
+is_a: MONDO:0700266 ! paraneoplastic cutaneous syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:409253]: add source

File: src/ontology/mondo-edit.obo [hunk 8]
```diff
@@ -495999,6 +496015,7 @@ xref: SCTID:47709007 {source="MONDO:equivalentTo"}
 xref: UMLS:C0149911 {source="MONDO:equivalentTo", source="NCIT:C3496"}
 is_a: MONDO:0001566 {source="MESH:C562390", source="NCIT:C3496"} ! hypercalcemia disease
 is_a: MONDO:0021073 {source="MESH:C562390"} ! paraneoplastic syndrome
+is_a: MONDO:0700253 ! paraneoplastic hematological syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:496018]: add source

File: src/ontology/mondo-edit.obo [hunk 9]
```diff
@@ -501799,6 +501816,7 @@ xref: GARD:22493 {source="Orphanet:623626"}
 xref: MESH:D020362 {source="https://orcid.org/0000-0003-1967-3726", source="MONDO:equivalentTo"}
 xref: NCIT:C4685 {source="MONDO:equivalentTo"}
 xref: Orphanet:623626 {source="MONDO:equivalentTo"}
+is_a: MONDO:0018215 ! paraneoplastic neurologic syndrome
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:501819]: add source

File: src/ontology/mondo-edit.obo [hunk 10]
```diff
@@ -517665,6 +517683,42 @@ xref: OMIM:225060 {source="MONDO:includedEntryInOMIM"}
...
+def: "Paraneoplastic syndrome that involves the endocrine system" [MONDO:patterns/location]
+is_a: MONDO:0021073 {source="PMID:29939667", source="https://orcid.org/0009-0001-6494-4831"} ! paraneoplastic syndrome
+intersection_of: MONDO:0021073 ! paraneoplastic syndrome
+intersection_of: disease_has_location UBERON:0000949 ! endocrine system
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7460" xsd:anyURI
+
+[Term]
+id: MONDO:0700253
+name: paraneoplastic hematological syndrome
+def: "Paraneoplastic syndrome that involves the hematopoietic system" [MONDO:patterns/location]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:517698]: need a period

File: src/ontology/mondo-edit.obo [hunk 11]
```diff
@@ -517665,6 +517683,42 @@ xref: OMIM:225060 {source="MONDO:includedEntryInOMIM"}
...
+is_a: MONDO:0021073 {source="PMID:29939667", source="https://orcid.org/0009-0001-6494-4831"} ! paraneoplastic syndrome
+intersection_of: MONDO:0021073 ! paraneoplastic syndrome
+intersection_of: disease_has_location UBERON:0002390 ! hematopoietic system
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7460" xsd:anyURI
+
+[Term]
+id: MONDO:0700254
+name: paraneoplastic gastrointestinal syndrome
+def: "Paraneoplastic syndrome that involves the digestive system" [MONDO:patterns/location]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:517707]: need a period

File: src/ontology/mondo-edit.obo [hunk 12]
```diff
@@ -517665,6 +517683,42 @@ xref: OMIM:225060 {source="MONDO:includedEntryInOMIM"}
...
+is_a: MONDO:0021073 {source="PMID:29939667", source="https://orcid.org/0009-0001-6494-4831"} ! paraneoplastic syndrome
+intersection_of: MONDO:0021073 ! paraneoplastic syndrome
+intersection_of: disease_has_location UBERON:0001007 ! digestive system
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7460" xsd:anyURI
+
+[Term]
+id: MONDO:0700255
+name: paraneoplastic renal syndrome
+def: "Paraneoplastic syndrome that involves the renal system" [MONDO:patterns/location]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:517716]: need a period

File: src/ontology/mondo-edit.obo [hunk 13]
```diff
@@ -517747,6 +517801,22 @@ xref: Orphanet:477647 {source="MONDO:equivalentTo"}
 is_a: MONDO:0019751 {source="PMID:37161741", source="https://orcid.org/0000-0002-4142-7153"} ! autoinflammatory syndrome
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7365" xsd:anyURI
 
+[Term]
+id: MONDO:0700265
+name: paraneoplastic rheumatic syndrome
+def: "Paraneoplastic syndrome that involves the joints, bones, muscles, and/or connective tissue" [MONDO:patterns/location]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:517807]: need a period

File: src/ontology/mondo-edit.obo [hunk 14]
```diff
@@ -517747,6 +517801,22 @@ xref: Orphanet:477647 {source="MONDO:equivalentTo"}
...
+is_a: MONDO:0021073 {source="PMID:29939667", source="https://orcid.org/0009-0001-6494-4831"} ! paraneoplastic syndrome
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7460" xsd:anyURI
+
+[Term]
+id: MONDO:0700266
+name: paraneoplastic cutaneous syndrome
+def: "Paraneoplastic syndrome that involves the integumental system" [MONDO:patterns/location]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:517814]: need a period

File: src/ontology/mondo-edit.obo [hunk 15]
```diff
@@ -517665,6 +517683,42 @@ xref: OMIM:225060 {source="MONDO:includedEntryInOMIM"}
 is_a: MONDO:0000358 {source="OMIM:225060"} ! orofacial cleft
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700252
+name: parneoplastic endocrine syndrome
+def: "Paraneoplastic syndrome that involves the endocrine system" [MONDO:patterns/location]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:517689]: need a period

**Review by nicolevasilevsky — APPROVED** [review]
looks good except a few minor edits are needed

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:335535]: done

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:342531]: done

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:377273]: done

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:377275]: done

File: src/ontology/mondo-edit.obo (hunk 5 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:391790]: done

File: src/ontology/mondo-edit.obo (hunk 6 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:400387]: done

File: src/ontology/mondo-edit.obo (hunk 7 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:409253]: done

File: src/ontology/mondo-edit.obo (hunk 9 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:501819]: done

File: src/ontology/mondo-edit.obo (hunk 15 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:517689]: done

File: src/ontology/mondo-edit.obo (hunk 10 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:517698]: done

File: src/ontology/mondo-edit.obo (hunk 11 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:517707]: done

File: src/ontology/mondo-edit.obo (hunk 12 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:517716]: done

File: src/ontology/mondo-edit.obo (hunk 13 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:517807]: done

File: src/ontology/mondo-edit.obo (hunk 14 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:517814]: done

File: src/ontology/mondo-edit.obo (hunk 8 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:496018]: done

**Pull request merged**